mandatory/3-function-output.js

```diff
@@ -2,15 +2,16 @@
 function getRandomNumber() {
   return Math.random() * 10;
 }
+// Function getRandomNumber between 0 and 1 and multiplies it by 10. //
 
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
   return word1.concat(word2);
 }
+//This function is used to connect strings together into one new string. //
 
 function concatenate(firstWord, secondWord, thirdWord) {
-  // Write the body of this function to concatenate three words together.
-  // Look at the test case below to understand what this function is expected to return.
+  return firstWord.concat(thirdWord);
```

Comment:
Are you including the secondWord parameter here? HINT: Have a look at what the test is expecting you to output. You don't necessarily need to use the **concat** method here.